plugins/woocommerce/includes/admin/class-wc-admin-settings.php

```diff
@@ -796,6 +799,10 @@ public static function save_fields( $options, $data = null ) {
 
 			// Loop options and get values to save.
 			foreach ( $options as $option ) {
+				if( isset( $option['name'] ) ) {
```

Comment:
We may wish to change this more significantly (see earlier note) but I'll quickly note the correct formatting here would be:

```suggestion
				if ( isset( $option['name'] ) ) {
```

Reply:
Yes I think you are right about adding a new variable, indeed the mutation of the `$option` array can be confusing
I will make the modification